mkpkg

```diff
@@ -159,6 +159,9 @@ class Functions:
         return sorted(deps)
 
     def mapfiles(self, src, dst, *files, **kwargs):
+        if not files:
+            die("'FUN.mapfiles()' expects at least one file to be passed")
+
```

Comment:
`FUN.mapfiles()` returns `[]` if it is called as:
```
FUN.mapfiles('path/to/src', 'path/to/dst')
```
Since calling it this way does not show any errors or warnings, it could very easily lead to the programmer assuming that it will return `path/to/src/=path/to/dst/`, while in reality it returns an empty list.

#69 set out to fix exactly this, but by actually making the function return `path/to/src/=path/to/dst/`.